terraform/hcl_mapping.go

```diff
@@ -0,0 +1,41 @@
+package terraform
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	gohcl "github.com/hashicorp/hcl/v2/gohcl"
+	hclparse "github.com/hashicorp/hcl/v2/hclparse"
+	structs "github.com/ministryofjustice/cloud-platform-go-library/structs"
+)
+
+func GetTFBody(source string) (hcl.Body, error) {
+	// Parse the HCL file
+	parser := hclparse.NewParser()
+	file, diags := parser.ParseHCLFile(source)
+	if diags.HasErrors() {
+		fmt.Println("error parsing HCL file")
+		return nil, fmt.Errorf("error parsing HCL file")
+	}
+	return file.Body, nil
+}
+
+func MapTFFile(source string, body hcl.Body) (interface{}, error) {
+	// switch case to map the terraform file to the struct
```

Comment:
Instead of this comment we can rename the function to describe the behaviour better and then delete the comment eg. `MapTfFileToStruct`